### PR TITLE
Add testing of semigroup presentations

### PR DESCRIPTION
**Pull request by SergeyKuz1001**

### CODE REVIEW

File: src/InGAP.hs [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE FlexibleInstances, FlexibleContexts #-}
+
+module InGAP where
```
Comment (gsvgit) [inline src/InGAP.hs:3]: May be "ToGAP"? Like "ToString", "ToList", other conversions to something.

File: src/SPTesting.hs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+module SPTesting where
```
Comment (gsvgit) [inline src/SPTesting.hs:1]: Can these functions be used for tests automation? For example, for unit tests.

File: src/InGAP.hs (hunk 1 above)
Comment (SergeyKuz1001) [reply src/InGAP.hs]: Ok, I've fix it

File: src/SPTesting.hs (hunk 2 above)
Comment (SergeyKuz1001) [reply src/SPTesting.hs:1]: I've created new module based on this module for unit tests

**Pull request merged**